internify/src/components/molecules/styles/InputFormContactDetails.css

```diff
@@ -0,0 +1,12 @@
+.input_contact_row {
+  margin-bottom: 3%;
+}
+
+.i-form-outer {
```

Comment:
this style is overriding styles from the first page of the job creation step. Make sure you're specifying class names more so that they don't override other pages. You can probably fix this by changing the css class to: `.input_contact_row .i-form-outer` or something. 

<img width="948" alt="image" src="https://user-images.githubusercontent.com/37882255/122346589-fb1d9600-cefd-11eb-8916-8d81db6cf057.png">

Reply:
nice catch Tommy!